WebPhapp/client/src/App/pages/Patient.js

```diff
@@ -6,12 +6,14 @@ import qs from 'qs';
 class Patient extends Component {
   // Initialize the state
   state = {
-    prescriptions: []
+    prescriptions: [],
+    isFetching: true
   };
 
   // Fetch the prescription on first mount
   componentDidMount() {
     this.getPrescriptions();
+    this.state.isFetching = false;
```

Comment:
Can we set the state of this inside of the axios promise within the getPrescriptions function? Otherwise, this could create  race condition.